src/material/chips/chips.md

```diff
@@ -1 +1,189 @@
-<!-- TODO: this guide and the related live examples need to be rewritten to reflect the new chips API. -->
+Chips allow users to view information, make selections, filter content, and enter data.
+
+### Static Chips
+
+Chips are always used inside a container. To create chips, start with a `<mat-chip-set>` element. Then, nest `<mat-chip>` elements inside the `<mat-chip-set>`.
+
+<!-- example(chips-overview) -->
+
+By default, `<mat-chip>` renders a chip with Material Design styles applied. For a chip with no styles applied, use `<mat-basic-chip>`.
+
+*Hint: `<mat-basic-chip>` receives the `mat-mdc-basic-chip` CSS class in addition to the `mat-mdc-chip` class.*
+
+#### Disabled appearance
+
+Although `<mat-chip>` is not interactive, you can set the `disabled` Input to give it disabled appearance.
+
+```html
+<mat-chip disabled>Orange</mat-chip>
+```
+
+### Selection Chips
+
+Use `<mat-chip-listbox>` and `<mat-chip-option>` for selecting one or many items from a list. Start with creating a `<mat-chip-listbox>` element. If the user may select more than one option, add the `multiple` attribute. Nest a `<mat-chip-option>` element inside the `<mat-chip-listbox>` for each available option.
+
+#### Disabled `<mat-chip-option>`
+
+Use the `disabled` Input to disable a `<mat-chip-option>`. This gives the `<mat-chip-option>` a disabled appearance and prevents the user from interacting with it.
+
+```html
+<mat-chip-option disabled>Orange</mat-chip-option>
+```
+
+#### Keyboard Interactions
```

Comment:
nit: not sure if we the keyboard interactions section shouldn't be part of the "Accessibility" section. Not a blocker though.